README.md

```diff
@@ -101,7 +101,7 @@ Clone this repository to your desired folder:
 
 ```sh
   cd my-folder
-  git clone https://github.com/Divinity-dev/To-do-list.git
+  git clone https://github.com/Divinity-dev/Leaderboard
 ```
 -
 
```

Comment:
- [ ] [OPTIONAL] I highly suggest you remove the lines starting with `> Describe...` from your README file to make it professional. They are just instructions on what to do there. 🥸️

**Example**

![Screenshot from 2023-02-09 15-47-59](https://user-images.githubusercontent.com/17243325/217817782-b35e5799-21d4-4faf-b062-e3c731c66aa2.png)
![Screenshot from 2023-02-09 15-48-34](https://user-images.githubusercontent.com/17243325/217817794-1b1c09fd-3ae5-4a5b-b441-964e9575c33d.png)

- [ ] [OPTIONAL] I think it would be great to add content to the `FAQ` section or remove it completely since it is an optional section. 😎️

![Screenshot from 2023-02-09 15-48-57](https://user-images.githubusercontent.com/17243325/217818234-4269ec92-31da-40d6-8bcd-7a2a083b05a4.png)